uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -237,21 +226,14 @@ def test_can_enable_false_on_unsupported_architecture(self, tmpdir):
         with mock.patch('uaclient.util.get_platform_info') as m_platform:
             with mock.patch('sys.stdout', new_callable=StringIO) as m_stdout:
                 m_platform.return_value = unsupported_kernel
-                entitlement = LivepatchEntitlement(
-                    entitlement.cfg)
+                entitlement = LivepatchEntitlement(entitlement.cfg)
```

Comment:
Good 1. This wasn't converted because it was throwing test errors at me due the the entitlement fixtture error you caught above. I had misinterpreted the errors and being a result of using the fixture  instead of that my common fixture setup was wrong (writing esm instead of livepatch). Fixed remaining cases so all use the fixture now.